components/list.css

```diff
@@ -0,0 +1,72 @@
+.unorder1 {
+    list-style-type: circle;
+}
+  
+.unorder2 {
+    list-style-type: square;
+}
+
+.unorder3 {
+    list-style-type: disc;
+}
+  
+.order1 {
+    list-style-type: decimal-leading-zero;
+}
+  
+.order2 {
+    list-style-type: upper-alpha;
+}
+
```

Comment:
maybe class name can be order2-list(feels more descriptive )

Reply:
Sure.